docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3.9"
+services:
+  postgres:
+    image: postgres:16.1-bookworm
+    ports:
+      - ${POSTGRES_PORT}:${POSTGRES_PORT}
+    restart: on-failure:5
+    env_file:
+      - .env
+    volumes:
+      - ./postgres:/var/lib/postgresql/data
```

Comment:
u can specify just volume name and then it would be not in current dir and thus no need to add to .gitignore